services/cargohold/src/CargoHold/S3.hs

```diff
@@ -500,15 +501,22 @@ completeResumable r = do
             throwE $ uploadIncomplete (resumableTotalSize r) total
 
     -- Construct a 'Source' by downloading the chunks.
-    chunkSource env s3c b cs = case Seq.viewl cs of
+    chunkSource :: Manager
+                -> Aws.Env
+                -> S3Configuration Aws.NormalQuery
+                -> Bucket
+                -> Seq S3Chunk
+                -> ConduitT () ByteString (ResourceT IO) ()
+    chunkSource man env s3c b cs = case Seq.viewl cs of
         EmptyL  -> mempty
         c :< cc -> do
-            (src, fin) <- lift $ do
+            -- (src, fin) <- lift $ do
```

Comment:
Why is this commented out? Is there some TODO that should've been done here?

Reply:
looks like it's safe to delete this.